lib/matplotlib/axes/_base.py

```diff
@@ -2917,12 +2917,12 @@ def set_xscale(self, value, **kwargs):
         """
         Set the x-axis scale
 
-        Set the scaling of the x-axis: %(scale)s
-
-        ACCEPTS: [%(scale)s]
+        Parameters
+        ----------
+        value: float
+            Value to scale x-axis by
 
-        Different kwargs are accepted, depending on the scale:
-        %(scale_docs)s
+        Different kwargs are accepted, depending on the scale
```

Comment:
I know @NelleV wants to remove this interpolation stuff, but right now you've reduced the information in the docstring. There needs to _at least_ be a cross-link to the place where the other keyword arguments are defined.